MergerService/Utils/TaskUtils.cs

```diff
@@ -125,14 +122,20 @@ public void UpdateCompletion(string jobId, string taskId, string? managerCallbac
             }
         }
 
+        public bool IsMaxAttemptsReached(int attempts)
```

Comment:
This can be `static`, `maxAttempts` is read straight from the configuration / env.

Reply:
we talked about it, it can;t be static as configuratio is injected (instance) and can't be defined as static